.vuepress/public/_redirects

```diff
@@ -1,84 +1,84 @@
 /references/dumps.html                              /reference/api/dumps.html
-/resources                                          /create/how_to
-/resources/about_storage.html                       /reference/under_the_hood/storage.html
+/resources                                          /learn/cookbooks/running_production.html
+/resources/about_storage.html                       /learn/advanced/storage.html
 /resources/sdks.html                                /learn/what_is_meilisearch/sdks.html
 /resources/comparison_to_alternatives.html          /learn/what_is_meilisearch/comparison_to_alternatives.html
-/resources/postman_collection.html                  /create/how_to/postman_collection.html
+/resources/postman_collection.html                  /learn/cookbooks/postman_collection.html
 /resources/contact.html                             /learn/what_is_meilisearch/contact.html
-/resources/tutorials                                /learn/getting_started
-/running-production                                 /create/how_to/running_production.html
-/resources/howtos                                   /create/how_to
-/instant-search-react-app                           /create/how_to/meilisearch_react.html
-/resources/howtos/digitalocean_droplet.html         /create/how_to/digitalocean_droplet.html
-/resources/howtos/search_bar_for_docs.html          /create/how_to/search_bar_for_docs.html
-/http2-ssl                                          /create/how_to/http2_ssl.html
+/resources/tutorials                                /learn/getting_started/quick_start.html
+/running-production                                 /learn/cookbooks/running_production.html
+/resources/howtos                                   /learn/cookbooks/running_production.html
+/instant-search-react-app                           https://blog.meilisearch.com/instant-search-react-app/
+/resources/howtos/digitalocean_droplet.html         /learn/cookbooks/digitalocean_droplet.html
+/resources/howtos/search_bar_for_docs.html          /learn/cookbooks/search_bar_for_docs.html
+/http2-ssl                                          /learn/cookbooks/http2_ssl.html
 
 # Move language
-/learn/advanced/language.html                       /reference/features/language.html
+/learn/advanced/language.html                       /learn/what_is_meilisearch/language.html
 
 # Move installation
-/reference/features/installation.html               /learn/getting_started/installation.html
+/reference/features/installation.html               /learn/getting_started/quick_start.html
 
 # Rename tutorials to getting started
 /learn/tutorials                                    /learn/getting_started
 /learn/tutorials/getting_started.html               /learn/getting_started/quick_start.html
-/learn/tutorials/whats_next.html                    /learn/getting_started/whats_next.html
+/learn/tutorials/whats_next.html                    /learn/getting_started/quick_start.html
```

Comment:
Would it be better to use `/learn/getting_started/quick_start.html#what-s-next` instead?